src/utils/optionsparser.cc

```diff
@@ -460,6 +460,24 @@ void IntOption::SetVal(OptionsDict* dict, const ValueType& val) const {
   dict->Set<ValueType>(GetId(), val);
 }
 
+int IntOption::ValidateIntString(const std::string& val) {  
```

Comment:
This function doesn't access any class data, so no need to make it a member function of a class. I suggest to make it a free-standing function in anonymous namespace instead (like 'EscapeMd()` above).